### ISSUE TITLE
Can't unlike posts

### ISSUE THREAD

**Issue by cinnamon-bun** [fraction/oasis#29]
**What version of this package are you using?**
2.9.7

**What happened?**
The like button adds likes but doesn't remove a like from a post that you've already liked.  Looking at the HTML, the button always submits a `voteValue` of `1`.

**What did you expect to happen?**
If you've previously liked a post, maybe it should submit a `voteValue` of `0`?

**Are you willing to submit a pull request to fix this bug?**
Nope ❤️ 

I am guessing it needs to query each post to know if you're amongst the likers?

**Comment by christianbundy** [comment]
Oh no! Thanks for taking the time to post this.

When I like a post the post is turning red and the `voteValue` is turning into 0. Those [should](https://github.com/fraction/oasis/blob/344d0cade98149ed3503e263ee1ea110a54b860a/src/pages/views/components/post.js#L62-L64) happen at the same time. Is the like "button" changing color correctly?

**Comment by christianbundy** [comment]
Btw the "are you willing to submit a pull request to fix this bug" feels weird and accusatory. Sorry I added that, I copied from another issue template but I don't like how that feels. If you have thoughts on how to make a productive issue template I'd merge any improvement.

**Comment by jedahan** [comment]
Maybe something like “Would you be interested in fixing this with me?”

**Comment by cinnamon-bun** [comment]
My like button doesn't turn red and doesn't have the "liked" class.  I tried:
- reloading the page after liking
- running against Patchwork's sbot and oasis's sbot

So it's not noticing that I've already liked it.  However the number of likes does increase by 1 the first time I click it, so a like is being created and I can see it in Patchwork afterwards.

Also doesn't matter if my like was created in Patchwork or Oasis, it doesn't make the heart icon red in Oasis.

**Comment by christianbundy** [comment]
Oh, could you upgrade your dependencies when you have a chance? I think `npm install` or `yarn install` should do the trick.

I think this is the same bug as #26. The code uses `ssb.id` exposed by ssb-client@4.7.8, so if you haven't installed dependencies I think you may still be on 4.7.7. My understanding is that `package.json` specifies `ssb-client@4.x.x`, so this should hopefully Just Work.

**Comment by cinnamon-bun** [comment]
That fixed it!

I'm using npm not yarn, so the checked-in `yarn.lock` wasn't being used.  I was stuck on `ssb-client@4.7.7` in my `package-lock.json` so deleting node_modules and re-installing wasn't helping.

I deleted node_modules AND package-lock.json and reinstalled.  I ended up with `ssb-client@4.7.9`.  Now the like button works, turns red, can be toggled on and off.

🎉 

Maybe bump ssb-client to `^4.7.9` in `package.json` to prevent this from biting anyone else?

(Tested using oasis@2.9.10)
